### ISSUE TITLE
ASCII for Foggy weather doesn't render on the Extra Monitor

### ISSUE THREAD

**Issue by NilaierMusic** [Shaosil/LethalCompanyMods-GeneralImprovements#48]
Not much i can say here, it looks like `FoggyAnimations` probably doesn't get loaded on the screen, since even replacing the animation with a new one doesn't bring it on the extra monitor.
![Lethal_Company_s4ZaFIqbpk](https://github.com/Shaosil/LethalCompanyMods-GeneralImprovements/assets/86863646/bcd538e3-0f5b-42cd-a341-e6eefa7f23c3)
Might be due to the fact that we don't have `LevelWeatherType.Foggy` referenced in `SceneHelper.cs` or something..?
```cs
                    // Change the animation we are currently referencing
                    _curWeatherAnimations = StartOfRound.Instance.currentLevel?.currentWeather switch
                    {
                        LevelWeatherType.None => WeatherASCIIArt.ClearAnimations,
                        LevelWeatherType.Rainy => WeatherASCIIArt.RainAnimations,
                        LevelWeatherType.Stormy => WeatherASCIIArt.RainAnimations,
                        LevelWeatherType.Flooded => WeatherASCIIArt.FloodedAnimations,
                        LevelWeatherType.Eclipsed => WeatherASCIIArt.EclipsedAnimations,
                        _ => new string[] { string.Empty }
                    };
 ```
 
 Adding `LevelWeatherType.Foggy => WeatherASCIIArt.FoggyAnimations,` seems to fix this issue.
 
![Lethal_Company_XDwr1Uqvko](https://github.com/Shaosil/LethalCompanyMods-GeneralImprovements/assets/86863646/28743f44-3434-44a7-8e60-085b799585ed)
Also, might be cool to have a Braille Font support and smaller font for a more detailed ASCII art.

**Comment by Shaosil** [comment]
Hahaha yep, I forgot to include the foggy in there - that will be fixed in the next release.

**Comment by Shaosil** [comment]
Fixed as of v1.0.24
